tests/test_runs/test_run.py

```diff
@@ -179,3 +183,31 @@ def test_to_from_filesystem_no_model(self):
         # assert default behaviour is throwing an error
         with self.assertRaises(ValueError, msg='Could not find model.pkl'):
             openml.runs.OpenMLRun.from_filesystem(cache_path)
+
+    def test_publish_with_local_loaded_flow(self):
+        """
+        Publish a run tied to a local flow after it has first been saved to
+         and loaded from disk.
+        """
+        model = Pipeline([
+            ('imputer', Imputer(strategy='mean')),
+            ('classifier', DummyClassifier()),
+        ])
+        task = openml.tasks.get_task(119)
+        run = openml.runs.run_model_on_task(
+            model=model,
+            task=task,
+            add_local_measures=False,
+            avoid_duplicate_runs=False,
+            upload_flow=False
+        )
+
+        cache_path = os.path.join(
```

Comment:
Could you please check the base test class? It creates a temporary directory for each unit test. It would be great if you could use that instead of creating your own directory.

Reply:
This follows [this existing test](https://github.com/openml/openml-python/blob/develop/tests/test_runs/test_run.py#L159). Why was the generated folder name used there then? Should it also be changed?

Reply:
That is an excellent question. I'm afraid that this was oversight by me in reviewing the other test. It would be great if you could change them both.

Reply:
As far as I can tell, you are referring to `self.workdir`? The `to_filesystem` method [requires the targeted directory to be empty](https://github.com/openml/openml-python/blob/develop/openml/runs/run.py#L137). For this reason, it seems best to create an new unique subdirectory of `self.workdir` as we are doing here (it looks like @janvanrijn designed it like this originally). Granted we don't necessarily need two levels of subdirectories, one should suffice. If you are not referring to `self.workdir`, please point me in the right direction.

Reply:
Yes, it is `self.workdir`. But actually, the test is executed in the workdir any way, so nevermind.